examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -106,6 +109,14 @@ class PairingCommand : public CHIPCommand,
             AddArgument("device-remote-port", 0, UINT16_MAX, &mRemotePort);
             AddArgument("pase-only", 0, 1, &mPaseOnly);
             break;
+        case PairingMode::AlreadyDiscovered:
+            AddArgument("skip-commissioning-complete", 0, 1, &mSkipCommissioningComplete);
+            AddArgument("setup-pin-code", 0, 134217727, &mSetupPINCode);
+            AddArgument("discriminator", 0, 4096, &mDiscriminator);
```

Comment:
Why does the "already discovered" case need a discriminator?  It shouldn't need a discriminator.

Reply:
I guess this is because we need to fill discriminator field in RendezvousParameters, but this field does not count if IP address if specified.